cypress/support/step_definitions/commonSteps.js

```diff
@@ -0,0 +1,37 @@
+import { asSelect, themeSelect, sizeSelect, subtextInput, heightInput, childrenTextArea, labelInput, labelHelpInput } from "../../locators/commonLocators";
+
+When('I set children to {string}', (text) => {
```

Comment:
I'm unclear what these steps are used for?

Reply:
This step set component field called 'children'
![image](https://user-images.githubusercontent.com/13675809/54540365-2e2bf480-4998-11e9-8589-5b2cc6b65ec1.png)

Reply:
I meant I was unclear how these are used, but I've looked into it a bit more and think I understand now. It matches the sentence against the feature spec.